lib/trento_web/views/cluster_view.ex

```diff
@@ -0,0 +1,45 @@
+defmodule TrentoWeb.ClusterView do
+  use TrentoWeb, :view
+
+  def render("clusters.json", %{clusters: clusters}) do
+    render_many(clusters, __MODULE__, "cluster.json")
+  end
+
+  def render("cluster.json", %{cluster: cluster}), do: cluster
+
+  def render("cluster_registered.json", %{cluster: cluster}) do
+    cluster
+    |> Map.from_struct()
+    |> Map.delete(:__meta__)
+    |> Map.delete(:tags)
+  end
+
```

Comment:
Shouldn't we have a `cluster_details_updated` view? We have it for the hosts
But I understand that it is not that simple in this case